packages/builder/src/components/userInterface/EventsEditor/EventPropertyControl.svelte

```diff
@@ -1,17 +1,76 @@
 <script>
   import { Button, Modal } from "@budibase/bbui"
-  import EventEditorModal from "./EventEditorModal.svelte"
   import { createEventDispatcher } from "svelte"
+  import { store } from "builderStore"
+  import { notifier } from "builderStore/store/notifications"
+  import EventEditor from "./EventEditor.svelte"
+  import BottomDrawer from "components/common/BottomDrawer.svelte"
+  import { automationStore } from "builderStore"
+
   const dispatch = createEventDispatcher()
 
   export let value
   export let name
 
-  let modal
+  let drawerVisible
+
+  function showDrawer() {
+    drawerVisible = true
+  }
+
+  const saveEventData = async () => {
+    // any automations that need created from event triggers
+    const automationsToCreate = value.filter(
+      action => action["##eventHandlerType"] === "Trigger Automation"
+    )
+    automationsToCreate.forEach(action => createAutomation(action.parameters))
+
+    dispatch("change", value)
+    notifier.success("Component actions saved.")
+  }
+
+  // called by the parent modal when actions are saved
+  const createAutomation = async parameters => {
```

Comment:
This stuff is just moved from the trigger automation work, since the event editor is no longer in a modal.